parser/feed/src/test/java/de/danoeh/antennapod/parser/feed/element/PodcastIndexTranscriptParserTest.java

```diff
@@ -0,0 +1,67 @@
+package de.danoeh.antennapod.parser.feed.element;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import static org.junit.Assert.assertEquals;
+
+import de.danoeh.antennapod.parser.feed.PodcastIndexTranscriptParser;
+import de.danoeh.antennapod.model.feed.Transcript;
+
+@RunWith(RobolectricTestRunner.class)
+public class PodcastIndexTranscriptParserTest {
+    private static String srtStr = "1\n"
+            + "00:00:00,000 --> 00:00:02,730\n"
+            + "John Doe: Promoting your podcast in a new\n\n\n"
+            + "00:00:02,730 --> 00:00:04,600\n"
+            + "way. The latest from PogNews.";
```

Comment:
The manual parsing code with `split` and stuff feels a bit fragile. To prevent future crashes, could you please also add some tests for malformed input, so it fails gracefully?

Reply:
added more tests